Make.inc

```diff
@@ -496,18 +496,20 @@ ifeq ($(DARWINVER_GTE11),0) # Snow Leopard specific configuration
 USEGCC := 1
 USECLANG := 0
 MACOSX_VERSION_MIN := 10.6
-OPENBLAS_TARGET_ARCH:=NEHALEM
-OPENBLAS_DYNAMIC_ARCH:=0
-USE_SYSTEM_LIBUNWIND:=1
-else
-ifeq ($(DARWINVER_GTE13),0) # Lion / Mountain Lion specific configuration
+OPENBLAS_TARGET_ARCH := NEHALEM
+OPENBLAS_DYNAMIC_ARCH := 0
+USE_SYSTEM_LIBUNWIND := 1
+else ifeq ($(DARWINVER_GTE13),0) # Lion / Mountain Lion specific configuration
 USEGCC := 0
 USECLANG := 1
 MACOSX_VERSION_MIN := 10.6
 else # Newer versions
 USEGCC := 0
 USECLANG := 1
+MACOSX_VERSION_MIN := 10.9
```

Comment:
We should probably just go to 10.9 across the board.

Reply:
All macOS versions 10.9 and above use 10.9 as the minimum with the exception of Apple silicon which uses 11.0. Are you proposing dropping support for macOS versions below 10.9 or something else?

Reply:
If our buildbots are running 10.9 or later, then we are effectively not supporting anything older anyways. Might as well make that explicit and shut down the warnings. But that should not distract from this PR.

Reply:
Makes sense. I've added a commit which drops support for versions earlier than 10.9